corehq/apps/export/models/new.py

```diff
@@ -748,13 +810,45 @@ class CaseExportInstance(ExportInstance):
     case_type = StringProperty()
     type = CASE_EXPORT
 
+    def __init__(self, *args, **kwargs):
+        super(CaseExportInstance, self).__init__(*args, **kwargs)
+        # Conditional default values for self.filters
+        created_default_filters = "filters" not in kwargs and (len(args) == 0 or "filters" not in args[0])
+        if created_default_filters:
+            self.filters.show_all_data = True
+
+    @classmethod
+    def wrap(cls, data):
+        if "filters" not in data:
+            data['filters'] = {'show_all_data': True}
+        return super(CaseExportInstance, cls).wrap(data)
+
     @classmethod
     def _new_from_schema(cls, schema):
         return cls(
             domain=schema.domain,
             case_type=schema.case_type,
         )
 
+    def get_filters(self):
+        if self.filters:
+            from corehq.apps.export.forms import CaseExportFilterBuilder
+            filter_builder = CaseExportFilterBuilder(
+                Domain.get_by_name(self.domain), get_timezone_for_domain(self.domain)
+            )
+            return filter_builder.get_filter(
+                self.filters.can_access_all_locations,
```

Comment:
nit: the return type of an array and the pluralized `filters` makes me think that this is an array object when it's not